packages/api-client/src/components/ApiClient/Request/RequestAuthentication/SecurityScheme.vue

```diff
@@ -303,9 +380,50 @@ const startAuthentication = (url: string) => {
             Reset
           </CardFormButton>
         </template>
+        <CardFormRows>
```

Comment:
Copied this section into the logic for the implicit flow as the `v-else` at the end was causing issues with rendering as it was previously acting as a fall-through case for both implicit and password. Can do refactoring if this is not desired.